File: trail1/3N + 1 수열/3n-plus-1-sequence.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int N = int.Parse(Console.ReadLine());
        int cnt = 0;
        while (N>1){
            if (N % 2 == 0) {N /= 2;}
            else {N = 3 * N +1;}
            cnt++;
        }
        Console.WriteLine(cnt);
    }
}

File: trail1/3N + 1 수열/3n-plus-1-sequence.c
#include <stdio.h>

int main() {
    int N;
    scanf("%d", &N);
    int cnt = 0;
    while (N > 1) {
        if (N % 2 == 0) {N /= 2;}
        else {N = 3 * N + 1;}
        cnt++;
    }
    printf("%d", cnt);
    return 0;
}